pkg/controller/che/che_controller.go

```diff
@@ -705,24 +704,34 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 	cheMultiUser := deploy.GetCheMultiUser(instance)
 
 	if cheMultiUser == "false" {
-		pvcStatus := deploy.SyncPVCToCluster(deployContext, deploy.DefaultCheVolumeClaimName, "1Gi", cheFlavor)
+		done, err := deploy.SyncPVCToCluster(deployContext, deploy.DefaultCheVolumeClaimName, "1Gi", cheFlavor)
 		if !tests {
-			if !pvcStatus.Continue {
+			if !done {
```

Comment:
I guess averywhere `!tests` and `!done` could be joined in the one `if !tests && !done`